proofs/clenvtac.ml

```diff
@@ -72,7 +72,7 @@ let clenv_refine ?(with_evars=false) ?(with_classes=true) clenv =
         Typeclasses.resolve_typeclasses ~filter:Typeclasses.all_evars
           ~fail:(not with_evars) clenv.env clenv.evd
       in
-      Typeclasses.make_unresolvables (fun x -> List.mem_f Evar.equal x evars) evd'
+      Typeclasses.make_unresolvables (fun x -> true) evd'
```

Comment:
I think this would be worth a comment explaining what we are doing.

Reply:
True